app/assets/javascripts/nidacfrontend.js

```diff
@@ -10,4 +10,15 @@ if (window.history && window.history.replaceState && typeof window.history.repla
   window.history.replaceState(null, null, window.location.href);
 }
 
-
+// automatically submit document upload on file select
+function onFileSelect() {
+  try{
+    // NF-403 - hide the form controls, display the spinner and disable the 'Continue' button before submitting
+    document.getElementById("upload-form").className = "govuk-!-display-none";
+    document.getElementsByClassName("hidden-spinner-row").item(0).className = "govuk-summary-list__row";
```

Comment:
I think the following should work, and de-couples this functionality from summary list class names
```suggestion
    document.querySelector(".hidden-spinner-row").classList.remove("govuk-!-display-none");
```

Reply:
Done